quires.sql

```diff
@@ -51,8 +51,7 @@ WHERE date_of_birth BETWEEN DATE '1990-01-01' AND DATE '2000-12-31'
 GROUP BY species;
 
 
-/* Vet clinic database: query multiple tables */
-
+/* Vet-clinic database: query multiple tables */
 
```

Comment:
- [ ] This query _**How many animals are there per species?**_ produced the wrong answer as observed in the screenshot. There are only two animals that are of the specie `Digimon`. Also, you will need to check your data and run this query again with the necessary correction and produce new screenshots. 


![image](https://user-images.githubusercontent.com/84156714/230433679-7e5ac300-fb13-40a2-bab6-8a433b7750d7.png)

Reply:
@bobb-Rob 
thanks for your good and valuable feedback. 
you mention a great thing I fixed the issue you can see it now,

![query How many animals are there per species](https://user-images.githubusercontent.com/108926875/230637684-7af611df-c1cc-4f12-a1ea-bd0c445f3886.JPG)